.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: go
+sudo: required
+go:
+  - 1.12.x
+env:
+  global:
+    - CGO_ENABLED="0"
+    - GO111MODULE="on"
+    - TMPDIR="$(echo -n $TRAVIS_BUILD_DIR/tmp)"
+    - UNIDOC_GS_BIN_PATH="/usr/bin/gs"
+    - UNIDOC_EXTRACT_FORCETEST="0"
+    - UNIDOC_E2E_FORCE_TESTS="0"
+    - UNIDOC_EXTRACT_TESTDATA=""
+    - UNIDOC_RENDERTEST_BASELINE_PATH=""
+    - UNIDOC_PASSTHROUGH_TESTDATA=""
+    - UNIDOC_ALLOBJECTS_TESTDATA=""
+    - UNIDOC_SPLIT_TESTDATA=""
+    - UNIDOC_EXTRACT_IMAGES_TESTDATA=""
+    - UNIDOC_JBIG2_TESTDATA=""
+    - UNIDOC_FDFMERGE_TESTDATA=""
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install -y git ghostscript
+  - go get golang.org/x/lint/golint
+  - go get -t ./...
+  - mkdir -p $TMPDIR
```

Comment:
Would it be possible to download the testdata here from a testdata repository?

Reply:
Yes, I can clone the repository in the $TMPDIR path and use it from there.